tests/e2e/testsuites/modify_volume_tester.go

```diff
@@ -68,48 +69,50 @@ func (modifyVolumeTest *ModifyVolumeTest) Run(c clientset.Interface, ns *v1.Name
 		By("modifying the pvc with invalid annotations")
 		attemptInvalidModification(c, ns, testVolume)
 	}
-
-	By("modifying the pvc")
-	modifyingPvc, _ := c.CoreV1().PersistentVolumeClaims(ns.Name).Get(context.TODO(), testVolume.persistentVolumeClaim.Name, metav1.GetOptions{})
-	if testType == VolumeModifierForK8s {
-		AnnotatePvc(modifyingPvc, parametersWithPrefix)
-	} else if testType == ExternalResizer {
-		vac, err := c.StorageV1alpha1().VolumeAttributesClasses().Create(context.Background(), &v1alpha1.VolumeAttributesClass{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      formatOptionMountPod.pod.Name,
-				Namespace: ns.Name,
-			},
-			DriverName: "ebs.csi.aws.com",
-			Parameters: modifyVolumeTest.ModifyVolumeParameters,
-		}, metav1.CreateOptions{})
-		framework.ExpectNoError(err)
-
-		vacName := vac.Name
-		modifyingPvc.Spec.VolumeAttributesClassName = &vacName
-	}
-
-	var updatedPvcSize resource.Quantity
-	if modifyVolumeTest.ShouldResizeVolume {
-		By("resizing the pvc")
-		updatedPvcSize = IncreasePvcObjectStorage(modifyingPvc, DefaultSizeIncreaseGi)
-	}
-
-	modifiedPvc, err := c.CoreV1().PersistentVolumeClaims(ns.Name).Update(context.TODO(), modifyingPvc, metav1.UpdateOptions{})
-	if err != nil {
-		framework.ExpectNoError(err, fmt.Sprintf("fail to modify pvc(%s): %v", modifyingPvc.Name, err))
-	}
-	framework.Logf("updated pvc: %s\n", modifiedPvc.Annotations)
-
-	By("wait for and confirm pv modification")
-	if testType == VolumeModifierForK8s {
-		err = WaitForPvToModify(c, ns, testVolume.persistentVolume.Name, parametersWithPrefix, DefaultModificationTimeout, DefaultK8sApiPollingInterval)
-	} else if testType == ExternalResizer {
-		err = WaitForVacToApplyToPv(c, ns, testVolume.persistentVolume.Name, *modifyingPvc.Spec.VolumeAttributesClassName, DefaultModificationTimeout, DefaultK8sApiPollingInterval)
-	}
-	framework.ExpectNoError(err, fmt.Sprintf("fail to modify pv(%s): %v", modifyingPvc.Name, err))
-	if modifyVolumeTest.ShouldResizeVolume {
-		err = WaitForPvToResize(c, ns, testVolume.persistentVolume.Name, updatedPvcSize, DefaultResizeTimout, DefaultK8sApiPollingInterval)
-		framework.ExpectNoError(err, fmt.Sprintf("fail to resize pv(%s): %v", modifyingPvc.Name, err))
+	if os.Getenv("TEST_VAC") == "True" {
```

Comment:
Rather than wrapping here, can we instead wrap here so it's clear the test is skipped? https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/c365bf646f2e8c0bc36259052869609661b3bf9c/tests/e2e/modify_volume.go#L152-L154

Also instead of checking for specifically `True` we should just do `!= ""` so that values like `yes` or `true` also work

Also also, can we add a comment clarifying why we're doing this, something like
```
// HACK: Only run VAC tests if TEST_VAC is set - these tests are being skipped in CI because Kubernetes 1.31 is available
// This check can safely be removed after the single-az test is moved to Kubernetes 1.31 or later

Reply:
will do now thank you !